final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import json_date_util as util
+import text_work_util as dt
+
+
+def parsed():
```

Comment:
It is better to use variable and function names that contain unambiguous answer to the question: `What is this?`